File: printntimes.py
def print_name(i, num):
    if i > num:
        return
    print("ALBIN")
    print_name(i + 1, num)

def main():
    n = int(input("Enter a number: "))
    print_name(1, n)

if __name__ == "__main__":
    main()

File: basic arithmetic/printntimes.cpp
#include <iostream>
using namespace std;

void print(int i,int num) {
    if(i>num){
        return;
    }
    cout<<"ALBIN";
    cout<<endl;
    print(i+1,num);
}

int main() {
    int n;
    cout << "Enter a number: ";
    cin >> n;
    print(1,n);
    return 0;
}
